Wikipedia/Code/UIAccessibility+Grouping.swift

```diff
@@ -0,0 +1,9 @@
+
+extension UIAccessibility {
```

Comment:
UIAccessibility is an informal protocol, and iirc its declaration only exists in Swift, so we can't use this static method in Obj-C, but thought this might not be a blocker? Also thought about hanging the method off of an Array extension (where the elements were optional strings), but that seemed a bit too leaky an abstraction...